src/Frequently-Asked-Questions.md

```diff
@@ -160,6 +160,26 @@ Please see our blog article on custom branding for your Element instance, here [
 
 You will be able to enter the customization preferences from the managed host page of your EMS account - [https://ems.element.io/user/hosting](https://ems.element.io/user/hosting).
 
+#### Can I use a subdomain instead of the root domain with my EMS server?
+
+Yes. However this is not recommended. For the same reason your email address probably is not `someone@email.example.com`, you probably don't want your Matrix IDs to be `@someone:matrix.example.com`.
+
+Please see <https://matrix-org.github.io/synapse/latest/setup/installation.html#choosing-your-server-name> for additional details on your server name.
+
+#### Can I use EMS hosted well-knowns with the root of my domain?
+
+Yes you can, however there are some limitations:
+
+- You will not be able to serve a website on the domain.
+- Using a CNAME DNS record on the root of a domain is not compliant with the DNS Specification (per [Domain Name System RFC 1034, paragraph 3.6.2](https://joinup.ec.europa.eu/collection/ict-standards-procurement/solution/dns-rfc-1034-rfc-1035-domain-name-system/about) specifically). But you may still be able to do this successfully if:
+  - Your DNS provider allows setting a CNAME record on the root of your domain. Be aware that certain other DNS records for your domain will not be returned properly, including SOA, NS, and TXT records. (Such as SPF, DMARC, and DKIM which is used for securing email)
```

Comment:
```suggestion
  - Your DNS provider allows setting a CNAME record on the root of your domain. Be aware that certain other DNS records for your domain will not be returned properly, including SOA, NS, and TXT records. (Such as SPF, DMARC, and DKIM which is used for securing email) OR
```
Add an `OR` to make it clear there are two alternatives if not using an A record: 
i) non spec compliant CNAME at root OR
ii) spec compliant ALIAS record or CNAME flattening